### PR TITLE
Add note about FF Actions API to docs, closes #TB-86

### PR DESCRIPTION
**Pull request by rogozinds**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/testbench/863)

### CODE REVIEW

**Comment by Artur-** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[documentation/testbench-quickstart.asciidoc, line 35 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxkXKVYqtGfsjQUYv:-KZzxkXKVYqtGfsjQUYw:batxq8y) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L35)):*
> ```AsciiDoc
> 
> [NOTE]
> Latest Firefox (version 50.1.0) does not support link:https://github.com/mozilla/geckodriver/issues/159[actions API].
> ```

The problem is not in Firefox but in Geckodriver, so maybe rephrase a bit e.g.
"Geckodriver used for the latest Firefox versions (48 and newer) does not support ..."

---

*[documentation/testbench-quickstart.asciidoc, line 36 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxe709yujGuuVmVAK:-KZzxe709yujGuuVmVAL:bx390wz) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L36)):*
> can be fixed
"has been fixed", otherwise it sounds like we are going to fix it

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/testbench/863)*

**Comment by rogozinds** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[documentation/testbench-quickstart.asciidoc, line 35 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxkXKVYqtGfsjQUYv:-K_-4bApaliEnN2pBH1Z:b-896fix) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L35)):*
<details><summary><i>Previously, Artur- (Artur) wrote…</i></summary><blockquote>

The problem is not in Firefox but in Geckodriver, so maybe rephrase a bit e.g.
"Geckodriver used for the latest Firefox versions (48 and newer) does not support ..."
</blockquote></details>

Done.

---

*[documentation/testbench-quickstart.asciidoc, line 36 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxe709yujGuuVmVAK:-K_-4_EO7RqA30EzH0eM:b-896fix) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L36)):*
<details><summary><i>Previously, Artur- (Artur) wrote…</i></summary><blockquote>

> can be fixed
"has been fixed", otherwise it sounds like we are going to fix it
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/testbench/863)*

**Comment by Artur-** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[documentation/testbench-quickstart.asciidoc, line 36 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxe709yujGuuVmVAK:-KZzx_NXzxwgAWxKt_83:byh4ekf) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L36)):*
<details><summary><i>Previously, rogozinds (rogozinds) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

"has been fixed"

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/testbench/863)*

**Comment by rogozinds** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[documentation/testbench-quickstart.asciidoc, line 36 at r1](https://reviewable.io:443/reviews/vaadin/testbench/863#-KZzxe709yujGuuVmVAK:-K_-bJKviA3L4w9oXBjk:b-896fix) ([raw file](https://github.com/vaadin/testbench/blob/ca25486018617088b593e18e3ede7aafe50c3c3e/documentation/testbench-quickstart.asciidoc#L36)):*
<details><summary><i>Previously, Artur- (Artur) wrote…</i></summary><blockquote>

"has been fixed"
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/testbench/863)*

**Comment by Artur-** [comment]
Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/testbench/863)*
